seed/views/v3/data_quality.py

```diff
@@ -0,0 +1,478 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+
+import csv
+
+from celery.utils.log import get_task_logger
+from django.http import JsonResponse, HttpResponse
+from rest_framework import viewsets, serializers, status
+from rest_framework.decorators import action
+from unidecode import unidecode
+
+from seed.data_importer.tasks import do_checks
+from seed.decorators import ajax_request_class
+from seed.lib.superperms.orgs.decorators import has_perm_class
+from seed.lib.superperms.orgs.models import (
+    Organization,
+)
+from seed.models.data_quality import (
+    Rule,
+    DataQualityCheck,
+)
+from seed.utils.api import api_endpoint_class
+from seed.utils.api_schema import AutoSchemaHelper
```

Comment:
This line and others defining or referencing the DataQualitySchema class are the only notable changes. The rest of the lines were just copied over.